modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -0,0 +1,77 @@
+= Create a Web Application Firewall Policy
+
+You can create a Web Application (WAF) policy in *Anypoint Security > Security Policies* to implement firewall security policy enforcement between your web application and client endpoint. WAF violation detection policies can also be escalated to Denial of Service (DoS) so that IPs and connections can be blocked.
+
+. Sign into Anypoint Platform and navigate to Anypoint Security.
+. In the Security Policies page, click *Create Policy* and select *Web Application Firewall (WAF)*.
+. Enter a name for the policy and click *Save*. +
+The policy appears in the Security Policies list, where you can edit and delete policies.
+
+image::security-policy-list.png[]
+
+== Configure Request Rulesets
+
+Configure the rules to allow or block request traffic to your Web application. For more information about request rulesets, see xref:waf-rulesets#request_rule_sets[Request rulesets].
+
+. In the menu on the left, click *Request Rulesets*. +
+All the rules are disabled by default. For each rule, you can: +
+* *Disable ruleset* - (Default) Ruleset detection is turned off.
+* *Detect and allow violations* - The violation is detected and you will get information in your log at the INFO level, per incident.
+* *Detect and reject violations* - The request is rejected and returns a response status of `400 - BAD REQUEST`.
++
+When you hover over the `i` to the right of each rule, the rule ID range for that rule is displayed. This information can be used for testing rulesets and for disabling individual rules.
```

Comment:
rule -> ruleset. 
Not sure how this information can be used for disabling individual rules.